### DISCUSSION TITLE
Updated ghostfolio but security token is not the same

### DISCUSSION THREAD

**Discussion by rubengr** [ghostfolio/ghostfolio#3832] [Q&A]
OK, i have already updated, but now doesn't make loggin with same security token. Maybe because i don't add in .env file.

Now i can't remember the REDIS password and POSTGRES password to add to .env file, in docker is possible to get it?

Thanks

_Originally posted by @rubengr in https://github.com/ghostfolio/ghostfolio/issues/3820#issuecomment-2376563023_

**Comment by rubengr** [comment]
Someone can help with this? @dtslvr

**Comment by Dilosen** [comment]
Hi 

make sure that you copy the code without any spaces. Sometimes you have a blank space at the end of the string.

If you haven't deleted the database, you should be able to access it again with the same security key
